### ISSUE TITLE
Lure time remaining is nonsense

### ISSUE THREAD

**Issue by hgentry** [AHAAAAAAA/PokemonGo-Map#2122]
After noticing that lures don't go away after they supposedly expire, but instead go away later seemingly at random, I decided to take a look at the code for this and it's not quite right.

There's a last_modified time and a current_time being used to determine when the lure expires, but there's no mention of the 30 minute timer anywhere. What I *think* is happening is it's calculating how long the lure has existed instead of how long it has left. Not certain. I think that section just has to be redone and I'm too busy to fix it myself.

**Comment by wcastello** [comment]
here? 

```
        var active_pokemon = idToPokemon[active_pokemon_id];

        var last_modified_date = new Date(last_modified);
        var current_date = new Date();

        var time_until_expire = current_date.getTime() - last_modified_date.getTime();

        var expire_date = new Date(current_date.getTime() + time_until_expire);
        var expire_time = expire_date.getTime();
```

**Comment by hgentry** [comment]
Yeah. It should be time_until_expire = 30 minutes - (now - last_modified)

**Comment by medghaim** [comment]
Known issue, fix identified, not sure where along the pipeline it is, but it'll be good for 2.1!

**Comment by Scyntrus** [comment]
https://github.com/AHAAAAAAA/PokemonGo-Map/blob/develop/static/js/map.js#L491

**Comment by Scyntrus** [comment]
Only way to determine true time remaining is to check when we first see an unlured stop change to a lured stop and add 27 minutes to its expiry time.
